src/wpool/mongoose_wpool_rdbms.erl

```diff
@@ -1,9 +1,13 @@
 -module(mongoose_wpool_rdbms).
 -behaviour(mongoose_wpool).
+-behaviour(mongoose_instrument_probe).
 
--export([init/0]).
--export([start/4]).
--export([stop/2]).
+-include("mongoose_metrics_definitions.hrl").
+
+-export([init/0, start/4, stop/2]).
+-export([probe/2, instrumentation/2]).
+
+-ignore_xref([instrumentation/2]). % optional wpool callback
```

Comment:
Hmm, I thought I saw an error here, and that's why I've added it - I was also surprised. But now I can't reproduce it, so it might have been something else. I will remove this line, thanks.